.circleci/config.yml

```diff
@@ -9,22 +9,34 @@ commands:
       - run:
           name: Test
           command: npm test
+  markdownlint:
+    steps:
+      - checkout
+      - run:
+        name: Install markdownlint
+        command: npm install markdownlint-cli
+      - run:
+        name: Markdownlint
+        command: markdownlint '**/*.md' --ignore node_modules
 jobs:
   node-v10:
     docker:
       - image: node:10
     steps:
       - test-node-id3
+      - markdownlint
```

Comment:
This does not seem to be taken into account (yet)? Do you have any idea why?

Reply:
Ah I can see an error in Circle CI now. `scrutinizer` seems to be a different tool.

Reply:
```text
ERROR IN CONFIG FILE:
[#/commands/markdownlint] 0 subschemas matched instead of one
1. [#/commands/markdownlint] expected type: String, found: Mapping
|   Command may be a string reference to another command
2. [#/commands/markdownlint/steps] 10 schema violations found
```

I might seem simply remove it, thoughts?